services/server/config/config.go

```diff
@@ -251,6 +274,32 @@ func loadConfigFile(path string) (*Config, error) {
 	return config, nil
 }
 
+func validaConfigUnsupported(path string) {
```

Comment:
```suggestion
func validateUnsupportedPluginProperties(b []byte) error {
```
The function should have a unit test to catch the error

Reply:
What about `validateUnsupportedNonPluginProperties`? Maybe too long, though